esx_service/cli/vmdkops_admin.py

```diff
@@ -857,8 +859,8 @@ def tenant_rm(args):
     # If args "remove_volumes" is not specified in CLI
     # args.remove_volumes will be None
     if args.remove_volumes:
-        if args.remove_volumes == 'True':
-            remove_volumes = True
+        print "All Volumes will be removed"
```

Comment:
Yes, but it deviated from the working state (on `print`, and on list iterators, and probably more) so another pass is required. And. more importantly, we need a CI with 6/5, and pylint as a part of the build. And coverage.py in test runs. So for now I will re-fix  on Py3 (or you can do it if you feel like it) and work with @kernetime to add pylinit to the build